ping.go

```diff
@@ -549,6 +553,12 @@ func (p *Pinger) processPacket(recv *packet) error {
 
 		outPkt.Rtt = receivedAt.Sub(timestamp)
 		outPkt.Seq = pkt.Seq
+		// if we've already received this sequence, ignore it.
```

Comment:
Nit, please make full sentences.

```suggestion
		// If we've already received this sequence, ignore it.
```